docs/source/api-docs/glowgr-functions.rst

```diff
@@ -0,0 +1,8 @@
+GloWGR
```

Comment:
The other high-level function API doc pages have titles ending with `Functions`, should we match that here?

Reply:
Since we have classes here as well, I don't think we should have `Functions`